portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Comment {
+    private final String text;
+    private final String user;
+    private final long timestamp;
+    private int upvotes;
+    private int downvotes;
```

Comment:
It seems like we might have gone in a few circles here. To be clear, it seems like these fields can be `final` because they do not appear to be modified anywhere in the application. Making them `final` will require removing the setter methods.

Reply:
got kind of confused before, fixed now!